lib/api/multiObjectDelete.js

```diff
@@ -255,6 +256,9 @@ function getObjMetadataAndDelete(authInfo, canonicalID, request,
         ], (err, objMD, deleteInfo, versionId) => {
             if (err === skipError) {
                 return moveOn();
+            } else if (err === objectLockedError) {
```

Comment:
Is the comparison valid?

Reply:
what do you mean? the error is returned as `objectLockedError` on line 233 above. and it passed manual testing :)

Reply:
What I meant was comparing error objects, I thought it would fail, but it works.

Reply:
Should work since `objectLockedError` is a singleton instance, it's always referring to the same object.